tasks/hidden_service.yml

```diff
@@ -4,7 +4,7 @@
 
 - name: Add the HiddenServiceDir config to torrc
   blockinfile:
-    dest: /etc/tor/torrc
+    dest: "{{ torrc }}"
```

Comment:
In the tor role (cf https://gitlab.com/osas/ansible-role-tor/merge_requests/3/diffs#diff-4 ). That's not great, but I can't think of a better way to get a cross role variable.